plugins/leemons-plugin-leebrary/frontend/src/request/hooks/keys/simpleAssetList.js

```diff
@@ -0,0 +1,14 @@
+export const allSimpleAssetListKey = [
+  {
+    plugin: 'plugin.leebrary',
+    scope: 'simpleAssetList',
+  },
+];
+
```

Comment:
Also add allGetSimpleAssetListKey.

But nicely implemented following the standard :D